pyproject.toml

```diff
@@ -2,16 +2,26 @@
 requires-python = ">=3.10"
 
 [tool.ruff]
-extend-exclude = ["scripts/util/version_check.py"]
+extend-exclude = [
+    # Exclude all third-party dependencies and environments.
+    # NOTE: Conda installs mgds+diffusers into "src/" in the project directory.
+    ".venv",
```

Comment:
I don't think these extra rules are needed. https://docs.astral.sh/ruff/configuration/#python-file-discovery
> By default, Ruff will also skip any files that are omitted via .ignore, .gitignore, .git/info/exclude, and global gitignore files 

If that doesn't work, can you add wildcards to the venv and .venv direcotries? It's already done in .gitignore

Reply:
@Nerogar It doesn't seem to work. Pre-commit was still including files in `src/` (mgds was being found by my linter when installed) even though our `.gitignore` has `/src` as a rule. Perhaps they just basically include each gitignore line as-is and don't properly understand our `/` prefix.

But yeah we can definitely wildcard the directories! Let's do that. Pushing in a moment.